synapse_span_table.py

```diff
@@ -277,6 +277,8 @@ def queue_span_table_record(self, tableName, data):
         try:
             spanTableDf = self.TABLE_QUEUES[tableName]
             spanTableDf = spanTableDf.append(df)
+            # fix for duplication problem
+            spanTableDf = spanTableDf.set_index(pd.Series(range(0,len(spanTableDf))))
```

Comment:
As I mentioned here [here](https://github.com/Sage-Bionetworks/synapsePythonClient/issues/867#issuecomment-862031955), using the ignore_index parameter to append should accomplish the same result in a slightly more succint way.

Reply:
Nice!